README.md

```diff
@@ -109,6 +110,35 @@ Stern will use the `$KUBECONFIG` environment variable if set. If both the
 environment variable and `--kubeconfig` flag are passed the cli flag will be
 used.
 
+## config file
+
+You can use the config file to change the default values of stern options. The default config file path is `~/.config/stern/config.yaml`. You can change it with `--config` flag or `STERNCONFIG` environment variable.
+
+The options available in the config file are:
+<!-- auto generated config file begin --->
+- `color`
+- `container-state`
+- `ephemeral-containers`
+- `exclude`
+- `exclude-container`
+- `exclude-pod`
+- `field-selector`
+- `include`
+- `init-containers`
+- `max-log-requests`
+- `namespace`
+- `only-log-lines`
+- `output`
+- `selector`
+- `since`
+- `tail`
+- `template`
+- `template-file`
+- `timestamps`
+- `timezone`
+- `verbosity`
```

Comment:
I am wondering if I should put them side by side.

> `color`, `container-state`, `ephemeral-containers`, `exclude`, `exclude-container`, `exclude-pod`, `field-selector`, `include`, `init-containers`, `max-log-requests`, `namespace`, `only-log-lines`, `output`, `selector`, `since`, `tail`, `template`, `template-file`, `timestamps`, `timezone`, `verbosity`